pom.xml

```diff
@@ -39,7 +39,7 @@
     </scm>
 
     <properties>
-        <jenkins.version>2.235.5</jenkins.version>
+        <jenkins.version>2.303.1</jenkins.version>
```

Comment:
Some Jenkins plugins were failing to start due to the version, that's why I upgraded it here.

Reply:
@chicofaccin I think that would be okay but I followed the recommendation here: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline

Reply:
@johnathafelix thanks. I didn't know about that. :)